### ISSUE TITLE
liferea-add-feed bug

### ISSUE THREAD

**Issue by Drakone** [lwindolf/liferea#479]
HI
Adding the RSS feed does not work.
```
[drakone@localhost ~]$ liferea-add-feed http://www.frandroid.com/android/applications/feed

(liferea:6702): libnotify-WARNING **: Failed to connect to proxy
/usr/bin/liferea-add-feed : ligne 25 :  6702 Erreur de segmentation  (core dumped)/usr/bin/liferea --add-feed "$NEWURL"
```

**Comment by Leiaz** [comment]
This is in Liferea 1.10 like #368 ? It is fixed in 1.12.

**Comment by Drakone** [comment]
You are right. Thanks.
